### PR TITLE
Update README.md to List out new library - OnlyPictures

### PR DESCRIPTION
**Pull request by KiranJasvanee**
Listed my overlapping circular pictures layout library.

Few months ago I've created one cool library of showing circular overlapping pictures which is popular now and eligible to take place in awesome-ios.

## Project URL
https://github.com/KiranJasvanee/OnlyPictures

## Category
UI

## Description
Listed this library in UI category

## Why it should be included to `awesome-ios` (mandatory)
This is the only kind of library with numerous flexibilities of lay-outing overlapping circular pictures. This is attractive and prevalent layout to show users now. It has to be in awesome-ios, so it can be reached to more ios developers.  

## Checklist

- [x] Only one project/change is in this pull request
- [ ] Has unit tests, integration tests or UI tests
- [x] Addition in chronological order (bottom of category)
- [x] Supports iOS 9 / tvOS 10 or later
- [x] Supports Swift 4 or later
- [x] Has a commit from less than 2 years ago
- [x] Has a **clear** README in English

### CODE REVIEW

**Comment by danger-awesome-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please update the Pull Request title to contain the library name</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Found 14 link issues, a project collaborator will take care of these, thanks :)</td>
    </tr>
  </tbody>
</table>

#### Link issues by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)

  Line | Status | Link
| ---: | :----: | --- |
206 | [403](https://httpstatuses.com/403) | https://www.udemy.com/arkit-beginner-to-professional/?couponCode=CREATORS 
407 | [301](https://httpstatuses.com/301) | https://github.com/cagricolak/CoreCharts redirects to<br>https://github.com/CoreCharts/CoreCharts
547 | [301](https://httpstatuses.com/301) | https://developer.couchbase.com/mobile/ redirects to<br>https://www.couchbase.com/products/mobile/
930 | [301](https://httpstatuses.com/301) | https://github.com/schibsted/layout redirects to<br>https://github.com/nicklockwood/layout
1003 | [301](https://httpstatuses.com/301) | https://github.com/caffe2/caffe2 redirects to<br>https://github.com/facebookarchive/caffe2
1174 | [301](https://httpstatuses.com/301) | https://github.com/bakkenbaeck/Viewer redirects to<br>https://github.com/3lvis/Viewer
1567 | [301](https://httpstatuses.com/301) | https://github.com/quickbirdstudios/RxCoordinator redirects to<br>https://github.com/quickbirdstudios/XCoordinator
1682 | [301](https://httpstatuses.com/301) | https://github.com/VirgilSecurity/sdk-x redirects to<br>https://github.com/VirgilSecurity/virgil-sdk-x
1705 | [301](https://httpstatuses.com/301) | https://github.com/VirgilSecurity/crypto-x redirects to<br>https://github.com/VirgilSecurity/virgil-crypto-x
2450 | [301](https://httpstatuses.com/301) | https://github.com/hyperoslo/Dropdowns redirects to<br>https://github.com/onmyway133/Dropdowns
2584 | [301](https://httpstatuses.com/301) | https://github.com/vsmithers1087/VSVerticalSlider redirects to<br>https://github.com/vsmithers1087/VerticalSlider
2706 | [301](https://httpstatuses.com/301) | https://github.com/bakkenbaeck/SectionScrubber redirects to<br>https://github.com/3lvis/SectionScrubber
3097 | [301](https://httpstatuses.com/301) | https://developer.apple.com/app-store/review/rejections/ redirects to<br>https://developer.apple.com/app-store/review/#common-app-rejections
3152 | [301](https://httpstatuses.com/301) | https://github.com/hyperoslo/SwiftSnippets redirects to<br>https://github.com/onmyway133/SwiftSnippets


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by KiranJasvanee** [comment]
Will it be done or still I need to fix something before uploading here?

**Comment by KiranJasvanee** [comment]
Any update?

**Comment by KiranJasvanee** [comment]
@vsouza Done. Thanks 👍

File: README.md [hunk 1]
```diff
@@ -1188,6 +1189,7 @@ Awesome-iOS is an amazing list for people who need a certain feature on their ap
 * [ATGMediaBrowser](https://github.com/altayer-digital/ATGMediaBrowser) - Image slide-show viewer with multiple predefined transition styles, and ability to create new transitions with ease.
 * [Pixel](https://github.com/muukii/Pixel) - An image editor and engine using CoreImage.
 
+
```
Comment (vsouza) [inline README.md]: Please, remove this blank line.

File: README.md [hunk 2]
```diff
@@ -1101,6 +1101,7 @@ Awesome-iOS is an amazing list for people who need a certain feature on their ap
 * [IDMPhotoBrowser](https://github.com/thiagoperes/IDMPhotoBrowser) - Photo Browser / Viewer.
 * [Concorde](https://github.com/contentful-labs/Concorde/) - Download and decode progressive JPEGs.
 * [TOCropViewController](https://github.com/TimOliver/TOCropViewController) - A view controller that allows users to crop UIImage objects.
+* [OnlyPictures](https://github.com/KiranJasvanee/OnlyPictures) - A simple and flexible way to add source of overlapping circular pictures, currently supports horizontal overlapping or distant pictures with great layout flexibility.
```
Comment (vsouza) [inline README.md]: Still in wrong position. You need to fix this and squash your commits, before I can merge

**Comment by vsouza** [comment]
I made some comments in your last commit. Please, fix the wrong changes and feel free to open PR once it is ready.

**Pull request closed without merging**